Rust/configs/alice_chess.toml

```diff
@@ -0,0 +1,13 @@
+chain = "00000000-0000-0000-0000-000000000000"
```

Comment:
we emphasize somewhere in the docs that this value **must be** generated as UUID (and why). This here is just a dummy value